application/android/src/co/theengine/loomplayer/LoomSensors.java

```diff
@@ -192,14 +187,15 @@ public void onSensorChanged(SensorEvent event)
                     ///if the sensor data is unreliable return
                     if(event.accuracy == SensorManager.SENSOR_STATUS_UNRELIABLE)
                     {
-                        return;
+                     //   return;
                     }
 
                     ///get correctly oriented values to work with
                     remappedValues = remapXYZValues(event.values);
-                    x = remappedValues[0];
-                    y = remappedValues[1];
-                    z = remappedValues[2];
+                    //normalize the values
+                    x = - remappedValues[0] / 10;
+                    y = - remappedValues[1] / 10;
+                    z = - remappedValues[2] / 10;
```

Comment:
This is done for consistency. Cocos had values in the range [-1,1] while android seems to have [10, -10] - thus the negate and divide.